Nanoleaf-wpf/Views/Options/OptionsWindow.xaml.cs

```diff
@@ -77,6 +79,27 @@ private void Cancel_Click(object sender, RoutedEventArgs e)
             Close();
         }
 
+        private void Visualize_Click(object sender, RoutedEventArgs e)
+        {
+            var screenMirrorAlgorithm = OptionsViewModel.ScreenMirrorAlgorithmMapping[OptionsViewModel.SelectedScreenMirrorAlgorithm];
+            
+            //Currently, only these 2 alorithm can be visualized
+            if (screenMirrorAlgorithm == ScreenMirrorAlgorithm.ScreenMirrorFit || screenMirrorAlgorithm == ScreenMirrorAlgorithm.ScreenMirrorStretch)
```

Comment:
and here would be the point to give a popup or such for ambilight not being able to display